### ISSUE TITLE
volume myVolume is already mounted on another host

### ISSUE THREAD

**Issue by zhengxiaochuan-3** [coreos/torus#435]
A torus Volume can not be mounted to another host ? Is this expected ?

**Comment by nak3** [comment]
Yes. There are similar requests as https://github.com/coreos/torus/issues/234 and https://github.com/coreos/torus/issues/235
